main.js

```diff
@@ -87,11 +87,12 @@ async function main() {
       body: resultsAsMarkdown,
     });
     core.info(
-      `Created comment id '${comment.id}' on issue '${contextObj.number}'.`
+      `Created comment id '${comment.id}' on issue '${contextObj.number}' in '${contextObj.repo}'.`
     );
     core.setOutput("comment-id", comment.id);
   } catch (err) {
     core.warning(`Failed to comment: ${err}`);
+    core.info(`Commenting is not possible from forks.`);
```

Comment:
No need to have a template here, it adds overhead:

```suggestion
    core.info('Commenting is not possible from forks.');
```

Reply:
Oh yeah, im not a js person. Actually didn't know what the backticks meant before this.